src/Validator/UserCredentialsValidator.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Smartsupp Live Chat integration module.
+ *
+ * @package   Smartsupp
+ * @author    Smartsupp <vladimir@smartsupp.com>
+ * @link      http://www.smartsupp.com
+ * @copyright 2016 Smartsupp.com
+ * @license   GPL-2.0+
+ *
+ * Plugin Name:       Smartsupp Live Chat
+ * Plugin URI:        http://www.smartsupp.com
+ * Description:       Adds Smartsupp Live Chat code to PrestaShop.
+ * Version:           2.2.0
+ * Author:            Smartsupp
+ * Author URI:        http://www.smartsupp.com
+ * Text Domain:       smartsupp
+ * License:           GPL-2.0+
+ * License URI:       http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+namespace Smartsupp\LiveChat\Validator;
+
+class UserCredentialsValidator
+{
+    const FILE_NAME = 'UserCredentialsValidator';
```

Comment:
netinka klases name perduot ar kazkaip? biski be rysio atrodo

Reply:
n moduliu taip padaryta, man ir be ryšio, bet savo funkcija atlieka neblogai